Sources/Remote Logging/Crash Logging/CrashLoggingDataProvider.swift

```diff
@@ -23,7 +23,13 @@ public protocol CrashLoggingDataProvider {
 public extension CrashLoggingDataProvider {
 
     var releaseName: String {
-        return Bundle.main.object(forInfoDictionaryKey: kCFBundleVersionKey as String) as! String
+        let bundleVersion = Bundle.main.infoDictionary?[kCFBundleVersionKey as String] ?? ""
+        let bundleIdentifier = Bundle.main.infoDictionary?[kCFBundleIdentifierKey as String] ?? ""
+        let bundleShortVersion = Bundle.main.infoDictionary?["CFBundleShortVersionString"] as? String ?? ""
+
+        // This is the format that Sentry recommends for version numbers.
+        // See https://docs.sentry.io/platforms/apple/configuration/releases/#bind-the-version
+        return "\(bundleIdentifier)@\(bundleShortVersion)+\(bundleVersion)"
     }
```

Comment:
Looking at the docs from that link, Sentry says it can set the value by itself if none is provided:

<img width="745" alt="image" src="https://github.com/Automattic/Automattic-Tracks-iOS/assets/1218433/f886d038-d346-47dd-bd96-d4f579baefa6">

Have you considered doing that?